docs/usage/beacon-management.md

```diff
@@ -33,7 +33,12 @@ You must generate a secret 32-byte (64 characters) hexadecimal string that will
 
 ### Configure Lodestar to locate the JWT secret
 
-When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwt-secret $JWT_SECRET_PATH` flag to point to the saved secret key file.
+When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwtSecret $JWT_SECRET_PATH` flag to point to the saved secret key file.
+
+### Set up and include identifiers in JWT tokens
+
+Lodestar auto-populates `clv` field in the claims of JWT authentication tokens with a non-configurable value `Lodestar/$CLIENT_VERSION` eg. `Lodestar/v1.3.0/2d0938e` to communicate the client's version. Lodestar also optionally includes `id` field in the claims with value `$JWT_ID` if the appropriate flag `--jwtId $JWT_ID` is added. 
+`id` and `clv` are particularly useful when running multiple consensus-layer clients with the different JWT secrets which makes the execution-layer client difficult to choose which JWT secret to verify against due to the inability to distinguish between the different consensus-layer clients. 
```

Comment:
> This might be confusing, as far as I am aware execution clients do not natively support multiplexing of consensus clients and I don't think this will ever be something that will be implemented. Without a multiplexer middleware like [eleel](https://github.com/sigp/eleel) it is not recommended to connect multiple CLs to a single EL.

I thought about the possible use case of configuring `--jwtId` for normal users and I can't think of any. The multiplexer case is nice but users who read this doc probably won't ever need it. 

I am inclined to just remove the part where it explains why `id` and `clv` are useful, and just emphasize that this flag exists and probably will not be used unless you are an advanced user and have a specific need to.

Man writing user doc is hard

Reply:
> The multiplexer case is nice but users who read this doc probably won't ever need it.

Yeah, I rather see people using eleel come to the CLI reference to see if Lodestar has the option to set the jwt id

> I am inclined to just remove this entirely because the point of this doc is to set up a beacon node. 

+1, I think should keep it simple here

> Man writing user doc is hard

Defintiely hard thing because it is quite subjective and user feedback is required in a lot of cases, @matthewkeil is working on a docs restructuring which should make it clearer on what details to include where.

In my opinion
- step-by-step instructions should mostly omit unnecessary details
- while sections explaining different concepts can go more into detail